webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder.js

```diff
@@ -817,7 +817,7 @@ function logout(cb){
 function initBsTooltips(){
   var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
   return tooltipTriggerList.map(function (tooltipTriggerEl) {
-    return new bootstrap.Tooltip(tooltipTriggerEl,{container : "body", html : true});
+    return new bootstrap.Tooltip(tooltipTriggerEl,{container : "body", html : true, trigger : 'hover'});
```

Comment:
By default, the trigger list is : `["hover", "focus"]`.
In some cases, the "focus" trigger can prevent certain tooltips from disappearing, so it is recommended not to use this trigger if this problem occurs.
However, this does not fix the problem of tooltips remaining displayed when the page is loading.